File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.py
class Solution(object):
    def maxProductDifference(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        n=len(nums)
        max1=nums[n-1]
        max2=nums[n-2]
        min1=nums[0]
        min2=nums[1]
        a=(max1*max2)-(min1*min2)
        return a

File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.java
class Solution {
    public int maxProductDifference(int[] b) {
       Arrays.sort(b);
       int n=b.length;
       int max1=b[n-1];
       int max2=b[n-2];
       int min1=b[0];
       int min2=b[1];
       int a=(max1*max2)-(min1*min2);
       return a;
        
    }
}